src/Http/Client/Response.php

```diff
@@ -60,11 +52,10 @@
  * $content = $response->getBody();
  * ```
  *
- * You can also use object access to get the string version
- * of the response body:
+ * You can get the body string using:
  *
  * ```
- * $content = $response->body;
+ * $content = $response->getBody()->getContents();
```

Comment:
This feels pretty clunky. I know its part of the spec, but I don't personally relish having to type that much to read a response's contents.

Reply:
We can add a `getStringBody()` method.